File: apps/mobile/src/config/env.ts
// src/config/env.ts — Centralized Environment & URL Configuration for SuprO Mobile
export const ENV = {
  // ─── DuckDNS Unified Endpoints ───
  API_URL: 'https://mysupro.duckdns.org',           // OCI Express Backend (Webhooks, WebSockets, APIs)
  CRM_URL: 'https://mysupro-crm.duckdns.org',       // Web CRM App & WebViews
  AUTH_URL: 'https://mysupro.duckdns.org',          // 100% OCI Backend (Zero Vercel)
  CDN_URL: 'https://mysupro-cdn.duckdns.org',       // Static CDN & APK Distribution

  // ─── WhatsApp Integration Numbers ───
  // User Configured: Route OTP requests via Admin Phone 916381029380
  WABA_PHONE: '916381029380',
  
  // Admin Support & Hotline (Course Guide, BDO Hotline, Human Support)
  ADMIN_PHONE: '916381029380',
  ADMIN_PHONES: ['6381029380', '916381029380'],
  
  // UPI Configuration
  ADMIN_UPI: '6381029380@hdfcbank',

  // Supabase Project Ref
  SUPABASE_URL: 'https://gmahjdzqitbomtmdzlfp.supabase.co',
  LMS_SUPABASE_URL: 'https://jjgdatjthyeesmgunnlp.supabase.co',
};

export default ENV;

File: apps/supro_flutter/lib/core/env.dart
// lib/core/env.dart — Centralized Environment & URL Configuration for SuprO Flutter
// Exact replica of apps/mobile/src/config/env.ts

class AppEnv {
  // ─── DuckDNS Unified OCI Endpoints ───
  static const String apiUrl = 'https://mysupro.duckdns.org';     // OCI Express Backend (Webhooks, WebSockets, APIs)
  static const String crmUrl = 'https://mysupro-crm.duckdns.org'; // Web CRM App & WebViews
  static const String authUrl = 'https://mysupro.duckdns.org';    // 100% OCI Backend (Zero Vercel)
  static const String cdnUrl = 'https://mysupro-cdn.duckdns.org'; // Static CDN & APK Distribution

  // ─── WhatsApp Integration Numbers ───
  static const String wabaPhone = '916381029380';
  static const String adminPhone = '916381029380';
  static const List<String> adminPhones = ['6381029380', '916381029380'];

  // ─── Payment Configuration ───
  static const String adminUpi = '6381029380@hdfcbank';

  // ─── Supabase Project Refs (Secondary Fallbacks) ───
  static const String supabaseUrl = 'https://gmahjdzqitbomtmdzlfp.supabase.co';
  static const String lmsSupabaseUrl = 'https://jjgdatjthyeesmgunnlp.supabase.co';
}
